src/media_devices/mod.rs

```diff
@@ -32,6 +36,19 @@ pub enum MediaDeviceInfoKind {
     AudioOutput,
 }
 
+pub(crate) fn create_device_id(
+    kind: u8,
+    host: String,
+    device_name: String,
+    num_channels: u16,
+    index: u8,
+) -> String {
+    let mut hasher = FxHasher::default();
```

Comment:
Instead of manually hashing it is probably best to include a private struct with DeviceInfo and derive Hash on it

Reply:
ok done